docs/guides/lint-arazzo.md

```diff
@@ -0,0 +1,73 @@
+# Lint Arazzo with Redocly CLI
+
+Arazzo is an open standard for describing multi-step API calls in sequence.
```

Comment:
Feedback on introduction:
- I know it's implied by the second sentence, but I think the relationship between Arazzo and OpenAPI is worth an early mention. 
- I'm not sure "multi-step API calls in sequence" conveys the right meaning and I think it undersells how great this standard is.

Reply:
This is brand new to a lot of folks, so we should tell them more about what it does.

I'll make an attempt. Like me, it's long-winded. So keep what you like and toss what you don't.

> Arazzo is an open standard for defining and orchestrating API workflows involving sequence of API calls. Arazzo supports interdependent API calls, where data from one request is used as input for a subsequent request. Originally developed with the working title of "OpenAPI Workflows", Arazzo extends the capabilities of existing API specifications by enabling them to describe multi-step, stateful interactions. The Redocly CLI offers built-in linting support for checking that your Arazzo files are valid.

Reply:
This isn't the Arazzo concept page though, it's just about how to lint the format. I did expand the intro, but only a bit.